pkg/os/terminal/terminal.go

```diff
@@ -0,0 +1,21 @@
+package terminal
+
+import (
+	"os"
+
+	xterminal "golang.org/x/term"
+)
+
+var (
+	// Global variable to force output regardless if terninal
+	ForceShowOutput bool = false
+)
+
+func IsShowTerminalOutput() bool {
+	// if this is a terminal or set to force output
+	return IsTerminal() || ForceShowOutput
+}
+
+func IsTerminal() bool {
+	return xterminal.IsTerminal(int(os.Stdout.Fd()))
```

Comment:
There's already a `RunningInTerminal` helper in `pkg/os/util.go`

Reply:
Already dealt with